tests/drivers/candev/main.c

```diff
@@ -33,6 +33,11 @@
 
 #include "periph/can.h"
 #include "can_params.h"
+#if defined(BOARD_SAME54_XPRO)
+#include "board.h"
+#include "periph_conf.h"
+#include "periph/gpio.h"
+#endif
```

Comment:
Why do you need this?

Reply:
In fact the `#if directive` can be removed, it's not needed. The includes are needed to define the CAN controller used by the SAME54-xpro